pkg/proxy/ipvs/proxier.go

```diff
@@ -1553,7 +1550,7 @@ func (proxier *Proxier) createAndLinkeKubeChain() {
 			if chain, ok := existingNATChains[ch.chain]; ok {
 				writeBytesLine(proxier.natChains, chain)
 			} else {
-				writeLine(proxier.natChains, utiliptables.MakeChainLine(kubePostroutingChain))
+				writeLine(proxier.natChains, utiliptables.MakeChainLine(ch.chain))
```

Comment:
compare code with iptables proxier, I think there is a mistake

Reply:
In this case `ch.chain` is not in `existingNATChains` so we fallback to `KUBE-POSTROUTING`. Doesn't seem like a bug to me (yet), but can you fix this in a separate PR please?

Reply:
ok, I will fix this in another PR.